src/components/WeddingDetails/WeddingDetails.tsx

```diff
@@ -24,23 +27,82 @@ const WeddingDetails: React.FC<IndividualWedding> = ({
 	photoList
 }) => {
 
+		const [detailsView, setDetailsView] = useState(true)
+		const [photoListView, setPhotoListView] = useState(false)
+		const [editListView, setEditListView] = useState(false)
+		const emailBody = `Dear ${name},
+			it is time to fill out your family photo list! Please follow the link provided to complete the missing photo information. Feel free to reach out if you have any questions.
```

Comment:
We can definitely look into formatting this email a little better if time allows, but this works great for now

Reply:
Agreed - nice to have in the future but at a lower priority than some other tasks